packages/library-base/src/commonMain/kotlin/io/realm/query/RealmQuery.kt

```diff
@@ -256,21 +257,54 @@ interface RealmSingleQuery<T> : Flowable<T> {
 
 /**
  * Queries that return scalar values. This type of query is used to more accurately represent the
- * results provided by some query operations, e.g. [RealmQuery.count] or [RealmQuery.min].
+ * results provided by some query operations, e.g. [RealmQuery.count] or [RealmQuery.sum].
```

Comment:
Isn't it only count that nonnull, sum also returns `null` if no elements are found?

Reply:
No, core returns 0 if the table is empty.

Reply:
I would probably question if that was a good idea, similar to how `average` shouldn't return 0, but probably not something we should fix in this PR